lib/server.js

```diff
@@ -189,3 +203,21 @@ module.exports = {
   }
 };
 
+function cleanUpAndExit() {
+  if (electronPids) {
```

Comment:
@cayter I just tried your fix and it was working for a while but now suddenly it has stopped working. I just get a message "[electron-connect] [server] killing electron process tree: pid" but the electron process never gets spawned after killing the old one. It only happens for the restart function. Did you face this issue?